packages/cli/src/index.ts

```diff
@@ -179,6 +180,11 @@ yargs
             description: 'Maximum execution time in seconds.',
             type: 'number',
           },
+          'ignore-deployment-failures': {
```

Comment:
Also it will need a docs update (in a different repo at the moment I know, it's fine to create the PR or a ticket on the monorepo and link to this PR for now)